AlfrescoApp/Model/Managers/RealmManager.m

```diff
@@ -51,7 +51,7 @@ - (void)createMainThreadRealmWithCompletionBlock:(void (^)(void))completionBlock
     if ([NSThread isMainThread])
     {
         dispatch_async(dispatch_get_main_queue(), ^{
-            _mainThreadRealm = [self createRealmWithName:[AccountManager sharedManager].selectedAccount.accountIdentifier];
+            _mainThreadRealm = [[RealmSyncCore sharedSyncCore] realmWithIdentifier:[AccountManager sharedManager].selectedAccount.accountIdentifier];
```

Comment:
Using a instance variable within a block creates a strong reference to self. Use a property instead with "weakify" and "strongify" constructs